### PR TITLE
[Milan] STRATEGIESSSSSSS

### PR DESCRIPTION
**Pull request by M-Byte480**

### CODE REVIEW

**Comment by M-Byte480** [comment]
LGTM

**Review by italotrt — APPROVED** [review]
lgtm

File: entities/department.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from .notification_entities import Email, SMS, Push
+
+class Department:
+    list_of_mediums = [Email, SMS, Push]
+
+    def __init__(self, name: str, medium_preference):
+        self.name = name
+        self.medium_preference = medium_preference
+        self.notification_medium = None
```
Comment (oOoOoOo0o) [inline entities/department.py:9]: Is this supposed to be a notification that takes a notification medium? And what is the purpose of medium_preference?

File: notification_entity/notification_medium.py
Comment (oOoOoOo0o) [inline notification_entity/notification_medium.py:1]: This is empty at the moment but what would be the difference between what would go here and notification_entities.py?

**Review by oOoOoOo0o — CHANGES_REQUESTED** [review]
Minor stuff really. Python syntax is making me feel queasy, not used to it.

**Pull request closed without merging**